modules/combined/test/tests/contact/horizontal_blocks_mortar_TM.i

```diff
@@ -0,0 +1,151 @@
+offset = 0.01
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  volumetric_locking_correction = true
+[]
+
+[Mesh]
+  [./original_file_mesh]
+    type = FileMeshGenerator
+    file = horizontal_blocks.e
```

Comment:
This could easily be done using mesh modifiers to save some repo space.

Reply:
Makes sense. I have switched to a generated mesh and deleted the original mesh file.